src/components/UserForm.js

```diff
@@ -151,7 +155,7 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
           autoComplete={
             variant === "signup" ? "new-password" : "current-password"
           }
-          placeholder="glitter-rainbow-butterfly-kitty"
+          placeholder="glittery-kitty-rainbowy-butterfly"
```

Comment:
I want the old one back.

Reply:
Se commit below.
Placeholder is now "glitter-rainbow-butterfly-kitty"

👍